src/languages/en.ts

```diff
@@ -1855,6 +1855,12 @@ export default {
             addRate: 'Add rate',
             workspaceDefault: 'Workspace currency default',
             foreignDefault: 'Foreign currency default',
+            value: 'Value',
```

Comment:
Where do we confirm this translation?

Reply:
I've asked once again for confirmation [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1710759109836829?thread_ts=1710156137.489269&cid=C01GTK53T8Q).